Arelle/ixbrl-viewer
# Add "disable-viewer" parameter.

#### Reason for change

Requested in #571 .

#### Description of change

A new `disable-viewer` parameter can be used to disable the viewer from loading so that the underlying iXBRL document can be seen directly.  If this parameter is present, and has any value other than `false`, the viewer will stop loading.

In stub viewer mode, this flag will cause the viewer to redirect to the iXBRL document (or the first iXBRL document, if there's more than one).  

#### Steps to Test

Add `?disable-viewer` to the URL for any document.  The viewer should not load, leaving you with the original iXBRL document loaded in the browser.

**review**:
@Arelle/arelle
@paulwarren-wk
